session.go

```diff
@@ -467,7 +469,10 @@ func (s *Session) mux(remoteBegin *frames.PerformBegin) {
 func (s *Session) muxFrameToLink(l *link, fr frames.FrameBody) {
 	select {
 	case l.RX <- fr:
+		// frame successfully sent to link
 	case <-l.Detached:
+		// link is closed
```

Comment:
Future: Depending on what type of frame this is we might want to disposition the message (ie, if it's a transfer). Like maybe we can .Release() it or similar.

Reply:
Looking at the code, I believe this case is actually impossible to hit.  In `muxDetach()`, `l.Detached` is closed after the link has been deallocated from the session, so there should be no way that the session mux can dispatch a frame to this link now.  Added a comment about this.